tailwind-plugins/button.js

```diff
@@ -51,7 +51,7 @@ module.exports = [
         ".btn": { ...btnDefault },
         ".btn-primary": {
           ...btnDefault,
-          border: "none",
+          border: "2px solid #595cf3",
           color: "#ffffff",
           backgroundColor: "#595cf3",
           borderColor: "transparent",
```

Comment:
Hi Simon! I think this line might be causing some issue. When I take a look at two buttons next to eachother, the primary button is still slightly smaller then secondary. However, when I look at browser tools I see that the border is still set to transparent. When I add 2px solid #595cf3 to through dev tools, they look to be the same size.

So it appears that this is the correction solution but something is causing it to be overriden!